receiver/faroreceiver/README.md

```diff
@@ -0,0 +1,21 @@
+# Faro Receiver
+
+This receiver can receive telemetry data from the [Grafana Faro Web SDK](https://github.com/grafana/faro-web-sdk).
+The telemetry data is in JSON format and adheres to the Faro OpenAPI schema, as defined in the [github.com/grafana/faro](https://github.com/grafana/faro)
+
+## Receiver Configuration
+
+The following receiver configuration parameters are supported.
```

Comment:
Need to reference the full available server config (https://github.com/open-telemetry/opentelemetry-collector/tree/main/config/confighttp) not just endpoint.

Reply:
Agreed